Create Api-Final Project.md (#62)
Laura-Novich-OBW/student-showcase: student-work/chana-willinger/Api-Final Project.md

## GET/tableNo

> This is the call sent when the server wants to get the bill. The GET request includes the table number. Take out orders are table 99. When this is printed, the customer can pay the bill.

Reviewer: ```suggestion
This is the call sent when the patron wants to get their bill. The GET request includes the table number. Take out orders are table 99. When this is printed, the customer can pay the bill.
```

## Code Example

> curl -X GET "http://URL/tableNo?id=99"

Reviewer: ```suggestion
```curl
      curl -X GET "http://URL/tableNo?id=99"
```      
```

## Response Example 

```JSON

{
   "orderNum":123,
   "timestamp":"2020-01-21T07:44:45-05:00",
   "Item1":{
      "ItemOrdered":{
         "type":"burgerMeal",
         "Cost":10.99
      }
   },
   "Item2":{
      "ItemOrdered":{
         "type":"salad",
         "Cost":9.50
      }

>    }

Reviewer: ```suggestion
   };
```
